app/uk/gov/hmrc/platformstatusfrontend/services/GarbageService.scala

```diff
@@ -28,25 +28,22 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.jdk.CollectionConverters._
 
 @Singleton
-class GarbageService @Inject()(backendConnector: BackendConnector)(implicit ec: ExecutionContext) {
+class GarbageService @Inject()(backendConnector: BackendConnector)(implicit ec: ExecutionContext):
 
   private val logger = Logger(this.getClass)
 
   def getBackendGcInfo(implicit hc: HeaderCarrier): Future[GcInformation] =
     backendConnector.gcInformation()
-      .recoverWith {
+      .recoverWith:
         case ex: Exception =>
           val msg = s"bodyToBackend call to backend service failed"
           logger.warn(msg, ex)
           Future.successful(GcInformation(-1, Seq[GarbageCollectorMXBean]()))
-      }
 
-  def getFrontendGcInfo: Future[GcInformation] = {
+  def getFrontendGcInfo: Future[GcInformation] =
```

Comment:
can the `()` be added to show this is a side-effecting function? (Same for the one above which only has an implicit parameter block)